server/src/immich/api-v1/asset/asset-repository.ts

```diff
@@ -260,13 +260,13 @@ export class AssetRepository implements IAssetRepository {
     asset.isArchived = dto.isArchived ?? asset.isArchived;
 
     if (asset.exifInfo != null) {
-      if (dto.description) {
```

Comment:
Yeah, that looks better. I think even just `dto.description !== undefined` would work/be better imo.